cse_site/ubs_project/models/test.py

```diff
@@ -0,0 +1,57 @@
+class Master(object):
```

Comment:
Does this file do anything with the website? or is this just test code that might should be excluded?